src/main/java/rx/internal/operators/OperatorFinally.java

```diff
@@ -33,6 +33,9 @@
     final Action0 action;
 
     public OperatorFinally(Action0 action) {
+        if (action == null) {
```

Comment:
The benefit of having the null check in the caller is that there is no allocation happening in case the action is null.